packages/integration-tests/utils/generatePlugin.ts

```diff
@@ -0,0 +1,124 @@
+import { Package } from '@sentry/types';
+import { readdirSync, readFileSync } from 'fs';
+import HtmlWebpackPlugin, { createHtmlTagObject } from 'html-webpack-plugin';
+import path from 'path';
+import { Compiler } from 'webpack';
+
+const PACKAGES_DIR = '../../packages';
+
+const tracingOnly = process.env.PW_TRACING_ONLY === 'true';
+const bundleKey = process.env.PW_BUNDLE;
+
+// `esm` and `cjs` builds are modules that can be imported / aliased by webpack
+const useCompiledModule = bundleKey === 'esm' || bundleKey === 'cjs';
+
+// Bundles need to be injected into HTML before Sentry initialization.
+const useBundle = bundleKey && !useCompiledModule;
+
+const BUNDLE_PATHS: Record<string, Record<string, string>> = {
+  browser: {
+    cjs: 'dist/index.js',
+    esm: 'esm/index.js',
+    bundle_es5: 'build/bundle.js',
+    bundle_es5_min: 'build/bundle.min.js',
+    bundle_es6: 'build/bundle.es6.js',
+    bundle_es6_min: 'build/bundle.es6.min.js',
+  },
+  tracing: {
+    cjs: 'dist/index.js',
+    esm: 'esm/index.js',
+    bundle_es5: 'build/bundle.tracing.js',
+    bundle_es5_min: 'build/bundle.tracing.min.js',
+    // `tracing` doesn't have an es6 build yet
+    bundle_es6: 'build/bundle.tracing.js',
+    bundle_es6_min: 'build/bundle.tracing.min.js',
+  },
+};
+
+/*
+ * Generate webpack aliases based on packages in monorepo
+ *
+ * When using compiled versions of the tracing and browser packages, their aliases look for example like
+ *     '@sentry/browser': 'path/to/sentry-javascript/packages/browser/esm/index.js'
+ * When using bundled versions of the tracing and browser packages, their aliases look for example like
+ *     '@sentry/browser': false
+ * so that the compiled versions aren't included
+ * All other packages are aliased for example like
+ *     '@sentry/serverless': 'path/to/sentry-javascript/packages/serverless'
+ */
```

Comment:
```suggestion
 * Generate webpack aliases based on packages in monorepo
 *
 * When using compiled versions of the tracing and browser packages, their aliases look for example like
 *     '@sentry/browser': 'path/to/sentry-javascript/packages/browser/esm/index.js'
 * and all other monorepo packages' aliases look for example like
 *     '@sentry/hub': 'path/to/sentry-javascript/packages/hub'
 *
 * When using bundled versions of the tracing and browser packages, all aliases look for example like
 *     '@sentry/browser': false
 * so that the compiled versions aren't included
 */
```

Updated to reflect the code change from the last code review.